docs/deployment/docker.rst

```diff
@@ -10,3 +10,12 @@ Then use Docker to build the image:
 .. code-block:: bash
 
  $ docker build -t mystorefront .
+
+
+Then you can run saleor container with the minimal settings:
```

Comment:
Let's capitalize the word "Saleor" in this sentence 😉

Reply:
Thanks!